File: Lista3/Python/Lcasa3/Ex10.py
while True:
    op = str(input("Digite: \n'+' para somar \n'-' para subtrair \n'*' para multiplicar \n'/' para dividir \n's' para sair \nOpção: "))
    if op != "s":
        a = int(input("Digite o primeiro número: "))
        b = int(input("Digite o segundo número: "))
        if op == "+":
            print(f"A soma de {a} e {b} = {a+b}")
        elif op == "-":
            print(f"A subtração de {a} e {b} = {a-b}")
        elif op == "*":
            print(f"A multiplicação de {a} e {b} = {a*b}")
        elif op == "/":
            print(f"A divisão de {a} por {b} = {a/b}")
        else:
            print("Opção inválida!")
        print("---------------------------------------------")
    else:
        break

File: Lista3/C/Lcasa3/Ex10.c
#include <stdio.h>

main()
{
	char op = 'n';
	int a, b;
	while(op != 's')
	{
		printf("Digite: \n'+' para somar \n'-' para subtrair \n'*' para multiplicar \n'/' para dividir \n's' para sair \nOpcao: ");
		scanf("%s", &op);
		if(op != 's')
		{
			printf("Digite o primeiro valor: ");
			scanf("%i", &a);
			printf("Digite o segundo valor: ");
			scanf("%i", &b);
			if(op == '+')
			{
				printf("A soma de %i e %i = %i\n", a, b, a+b);
			}
	        else if(op == '-')
	        {
	        	printf("A subtracao de %i e %i = %i\n", a, b, a-b);
			}
	        else if(op == '*')
	        {
	        	printf("A multiplicacao de %i e %i = %i\n", a, b, a*b);
			}
	        else if(op == '/')
	        {
	        	printf("A divisao de %i epor%i = %i\n", a, b, a/b);
			}
	        else
	        {
	        	printf("Opcao invalida!\n");
			}
	        printf("---------------------------------------------\n");
		}
		else
		{
		}
	}
}

